deps/mbedtls.mk

```diff
@@ -55,6 +58,12 @@ ifeq ($(OS), WINNT)
 else
 	$(call make-install,mbedtls-$(MBEDTLS_VER),)
 endif
+	$(INSTALL_NAME_CMD)libmbedx509.$(SHLIB_EXT) $(build_shlibdir)/libmbedx509.$(SHLIB_EXT)
+	$(INSTALL_NAME_CMD)libmbedtls.$(SHLIB_EXT) $(build_shlibdir)/libmbedtls.$(SHLIB_EXT)
+	$(INSTALL_NAME_CHANGE_CMD) libmbedx509.0.dylib @rpath/libmbedx509.$(SHLIB_EXT) $(build_shlibdir)/libmbedtls.$(SHLIB_EXT)
+	$(INSTALL_NAME_CHANGE_CMD) libmbedcrypto.0.dylib @rpath/libmbedcrypto.$(SHLIB_EXT) $(build_shlibdir)/libmbedtls.$(SHLIB_EXT)
+	$(INSTALL_NAME_CHANGE_CMD) libmbedcrypto.0.dylib @rpath/libmbedcrypto.$(SHLIB_EXT) $(build_shlibdir)/libmbedx509.$(SHLIB_EXT)
+	$(INSTALL_NAME_CMD)libmbedcrypto.$(SHLIB_EXT) $@
 	touch -c $(MBEDTLS_OBJ_TARGET)
```

Comment:
Doesn't make too much of a difference since this library is the one used for makefile dependencies.